cmd/ddev/cmd/debug-message-conditions.go

```diff
@@ -0,0 +1,38 @@
+package cmd
+
+import (
+	"github.com/ddev/ddev/pkg/config/remoteconfig"
+	"github.com/ddev/ddev/pkg/output"
+	"github.com/ddev/ddev/pkg/styles"
+	"github.com/jedib0t/go-pretty/v6/table"
+	"github.com/spf13/cobra"
+)
+
+// DebugMessageConditionsCmd implements the ddev debug message-conditions command
+var DebugMessageConditionsCmd = &cobra.Command{
+	Use:   "message-conditions",
+	Short: "Show message conditions of this version of ddev",
+	Run: func(cmd *cobra.Command, args []string) {
+		conditions := remoteconfig.ListConditions()
+
+		t := table.NewWriter()
+		styles.SetGlobalTableStyle(t)
+
+		t.AppendHeader(table.Row{"Condition", "Description"})
+
+		for condition, description := range conditions {
+			t.AppendRow(table.Row{condition, description})
+		}
+
+		t.SortBy([]table.SortBy{
+			{Name: "Condition", Mode: table.Asc},
+		})
+
+		output.UserOut.WithField("raw", conditions).Print("Supported conditions for messages are:\n\n", t.Render())
+	},
+	Hidden: true,
```

Comment:
I like that you hid this. The only thing is how will we ever find it!

Reply:
Therefor it's mentioned in the dev docs see https://github.com/ddev/ddev/pull/4885/files#diff-e7209a6b4e94d84c947308700da7f20a6190c6b8598adb0a0e119e58d9224b90R84